src/shared/components/palette/datetime-picker/datetime-picker.component.html

```diff
@@ -5,17 +5,22 @@
       <span class="form-label" *ngIf="caseField.label">{{caseField | ccdFieldLabel }}</span>
       <span class="form-hint" *ngIf="caseField.hint_text">{{caseField.hint_text}}</span>
       <span class="error-message"
-            *ngIf="dateControl && dateControl.errors && dateControl.touched">{{dateControl.errors | ccdFirstError}}</span>
+            *ngIf="dateControl && dateControl.errors && dateControl.touched && !(this.minError || this.maxError)">{{dateControl.errors | ccdFirstError:caseField.label}}</span>
```

Comment:
Not a regression as it's the same in `master` but this should definitely be using `dirty` and not `touched`.